### DISCUSSION TITLE
Release cadence?

### DISCUSSION THREAD

**Discussion by Quafadas** [scala-js/scala-js-dom#873] [General]
Two questions

Is there any particular criteria or trigger for scala-js-dom to release? I can see it's been a year or so since the last release. Although I'm not rushed and not out to pressure anyone, I'm trying to contribute to scala-dom-types, which would be (at least in part dependent on a 
release here). 

I recall seeing somewhere that the project publishes nightlies? I can't recall how to get at them ... but I think I could use them at least to satisfy the CI of scala-dom-types, whilst waiting for a release if that recollection is correct? 

@raquo

**Comment by raquo** [comment]
Looks like the nightlies are here: https://oss.sonatype.org/content/repositories/snapshots/org/scala-js/scalajs-dom_sjs1_3/ (found in scala-js discord)

I guess we can start compiling against that for now.

**Reply by Quafadas** [reply]
Huh, good spot, I clearly failed to operate discord correctly-  I thought I searched there - thank you 🙏
